app/routes/planet_routes.py

```diff
@@ -0,0 +1,134 @@
+from flask import Blueprint, abort, make_response, request, Response
+from app.models.planet import Planet
+from ..db import db
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.post("")
+def create_planet():
+    request_body = request.get_json()
+    name = request_body["name"]
+    description = request_body["description"]
+    num_moons = request_body["num_moons"]
+    
+    new_planet = Planet(name=name, description=description, num_moons=num_moons)
+    db.session.add(new_planet)
+    db.session.commit()
+
+    response = {
+        "id": new_planet.id,
+        "name": new_planet.name,
+        "description": new_planet.description,
+        "num_moons": new_planet.num_moons
+    }
```

Comment:
This is a place where you could call that method, like so `new_planet.to_dict()`.